src/spirc.rs

```diff
@@ -57,7 +65,7 @@ impl SpircManager {
             name: name,
             ident: ident,
             device_type: 5,
-            can_play: true,
+            can_play: player_is_some,
```

Comment:
How do other devices see librespot when `can_play` is false ?
I remember it being grayed out but still in the list (ie locked phone).

I think you need to remove the `kCanBePlayer` capability from the `DeviceState` for it to not appear in the list.